it/controllers/employment/EmploymentDetailsAndBenefitsControllerISpec.scala

```diff
@@ -0,0 +1,472 @@
+          )),
+          employmentBenefits = benefitsModel
+        )
+      ),
+      hmrcExpenses = None,
+      customerEmploymentData = Seq(),
+      customerExpenses = None
+    )
+  }
+
+
+  "as an individual in English" when {
+
+    ".show" should {
+
+      "return an action with benefits status Cannot update when no benefits data is in session" which {
+
+        lazy val playSessionCookies = PlaySessionCookieBaker.bakeSessionCookie(Map(
+          SessionValues.TAX_YEAR -> taxYear.toString,
+          SessionValues.CLIENT_NINO -> "AA123456A",
+          SessionValues.CLIENT_MTDITID -> "1234567890",
+          SessionValues.EMPLOYMENT_PRIOR_SUB -> Json.prettyPrint(Json.toJson(fullModel(None))
+          )))
+
+        lazy val result: WSResponse = {
+          authoriseIndividual()
+          await(wsClient.url(url).withHttpHeaders(HeaderNames.COOKIE -> playSessionCookies, "Csrf-Token" -> "nocheck").get())
+        }
+
+
+        implicit def document: () => Document = () => Jsoup.parse(result.body)
+
+        titleCheck(ContentEN.titleExpected)
+        h1Check(ContentEN.h1Expected)
+        captionCheck(ContentEN.captionExpected)
+
+        textOnPageCheck(ContentEN.p1ExpectedIndividual, p1Selector)
+
+        "has an employment details section" which {
+          linkCheck(ContentEN.fieldNames(0), employmentDetailsLinkSelector, employmentDetailsUrl)
+          textOnPageCheck("Updated", taskListRowFieldAmountSelector(1))
+        }
+
+        "has a benefits section" which {
+          textOnPageCheck(ContentEN.fieldNames(1), taskListRowFieldNameSelector(2))
+          textOnPageCheck("Cannot update", taskListRowFieldAmountSelector(2))
+        }
+        buttonCheck(ContentEN.buttonText, buttonSelector)
+
+        welshToggleCheck(ENGLISH)
+      }
+
+      "return an action with benefits status Updated when benefits data is in session" which {
+
+        lazy val playSessionCookies = PlaySessionCookieBaker.bakeSessionCookie(Map(
+          SessionValues.TAX_YEAR -> taxYear.toString,
+          SessionValues.CLIENT_NINO -> "AA123456A",
+          SessionValues.CLIENT_MTDITID -> "1234567890",
+          SessionValues.EMPLOYMENT_PRIOR_SUB -> Json.prettyPrint(Json.toJson(fullModel(Some(EmploymentBenefits("2020-04-04T01:01:01Z",Some(BenefitsModel)))
+          )))))
+
+        lazy val result: WSResponse = {
+          authoriseIndividual()
+          await(wsClient.url(url).withHttpHeaders(HeaderNames.COOKIE -> playSessionCookies, "Csrf-Token" -> "nocheck").get())
+        }
+
+
+        implicit def document: () => Document = () => Jsoup.parse(result.body)
+
+        titleCheck(ContentEN.titleExpected)
+        h1Check(ContentEN.h1Expected)
+        captionCheck(ContentEN.captionExpected)
+
+        textOnPageCheck(ContentEN.p1ExpectedIndividual, p1Selector)
+
+        "has an employment details section" which {
+          linkCheck(ContentEN.fieldNames(0), employmentDetailsLinkSelector, employmentDetailsUrl)
```

Comment:
i = 0 can be replaced with fieldNames.head. This will remove some of the warnings